ersilia/core/tracking.py

```diff
@@ -3,26 +3,85 @@
 import sys
 import csv
 import json
+import time
 import boto3
+import docker
 import shutil
+import psutil
 import logging
 import requests
 import tempfile
+import resource
 import statistics
 import tracemalloc
+from .session import Session
 from datetime import datetime
 from .base import ErsiliaBase
 from collections import defaultdict
 from ..default import EOS, ERSILIA_RUNS_FOLDER
-from ..io.output_logger import TabularResultLogger 
+from ..io.output_logger import TabularResultLogger
 from botocore.exceptions import ClientError, NoCredentialsError
 
 
-# Temporary path to log files until log files are fixed
-TEMP_FILE_LOGS = os.path.abspath("")
+def docker_stats(container_name=None):
```

Comment:
I think the `tracking.py` module is growing phenomenally large so we need to move this code to a more suitable module. I recommend moving it to https://github.com/ersilia-os/ersilia/blob/master/ersilia/utils/docker.py where you create two new methods in the `SimpleDocker` class as follows: `container_memory`, and `container_cpu`, and then use those here. And then you can create an object of this class in RunTracker and use these methods. Since the class doesn't really rely on any initial conditions, this should be straightforward.